### DISCUSSION TITLE
MLIR Dialects in Mojo SDK

### DISCUSSION THREAD

**Discussion by leandrolcampos** [modular/modular#692] [Q&A]
Which MLIR dialects are available in the current version of Mojo SDK? Are these dialects the same as those available in the Playground? Or now I can use `tensor` and `tosa` dialects to implement functions that take Mojo `Tensor`'s as arguments (like `gather`, `select`, and `while_loop`)?

**Comment by AleksandrTaranzhin** [comment]
> Which MLIR dialects are available in the current version of Mojo SDK? Are these dialects the same as those available in the Playground? Or now I can use `tensor` and `tosa` dialects to implement functions that take Mojo `Tensor`'s as arguments (like `gather`, `select`, and `while_loop`)?

****
